test/Macros/macros_diagnostics.swift

```diff
@@ -220,3 +220,8 @@ struct SomeType {
 macro multipleFreestandingRoles<T>(_: T) -> Void = #externalMacro(module: "A", type: "B")
 // expected-warning@-1{{external macro implementation type}}
 // expected-error@-2{{macro can only have a single freestanding role}}
+
```

Comment:
May I suggest having an additional test with the attributes in the opposite order too?